setup.php

```diff
@@ -256,6 +258,36 @@ function plugin_genericobject_includeCommonFields($force = false) {
    }
 }
 
+
+function plugin_genericobject_includeCommonFieldsLocale($force = false) {
+   global $CFG_GLPI,$LANG;
```

Comment:
```suggestion
```

Reply:
I can't remember at the moment, but I don't think it works properly on my installations without it.